README.md

```diff
@@ -345,6 +345,16 @@ the attribute type does not have any special semantic or effect currently.
 As already mentioned, all this configuration parameters can also be adjusted using the
 [`config.js`](https://github.com/telefonicaid/IoT-STH/blob/develop/config.js) file whose contents are self-explanatory.
 
+It is important to note that there is a limitation on the bytes a MongoDB namespace (concatenation of the database name and
```

Comment:
In addition, the last case is not about `collection-per-entity` but `collection-per-attribute`... so I reviewed it completely bad xD